dev/bots/test/analyze-test-input/root/packages/foo/stopwatch_fail.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This should fail analysis.
+
+void main() {
+  Stopwatch();
+
+  // Identify more than one in a file.
+  final Stopwatch
```

Comment:
is this supposed to be broken code or should this be something like

```suggestion
 Stopwatch myStopwatch;
```